### ISSUE TITLE
[Staging] Glyphs are placed an octave too high on the staff

### ISSUE THREAD

**Issue by JoyfulGen** [DDMAL/Rodan#1216]
Here's a demo where you can see that the glyphs are appearing an octave above where they should be. It's a bit messy, so I've put boxes around places where the issue is clear:

<img width="468" alt="Glyphs octave too high" src="https://github.com/user-attachments/assets/93405875-bb2e-424b-9d67-7d9184b9e9ed">

I suspect this is related to a fix to the Heuristic_Pitch_Finding job to make the default octave of clefs C4 instead of C3, because it looks like the pitches of the neumes have been fixed, but not the clefs. When staves don't have clefs, as in the example above, Neon assigns them a default C clef on the third line of the staff (from the bottom). That clef is still C3, but the neumes are now pitched an octave up, so they appear an octave too high. @lucasmarchd01, is this possible?

**Comment by lucasmarchd01** [comment]
Right, thanks for pointing this out! I think what's happening here is that previously in Neon, we would account for the incorrect clef location by adjusting the neume pitches an octave up. Now that the clef octave is correct, the pitches now appear an octave too high. Are you able to confirm that the MEI file output is correct? 

Then, I think this becomes a Neon issue for the correct display. @yinanazhou might be able to help with this?

**Comment by yinanazhou** [comment]
Yesss, this is related to https://github.com/DDMAL/Neon/issues/1241. I will push the changes this weekend. And Rodan needs to be updated to point to the that commit.

**Comment by kyrieb-ekat** [comment]
Note on this; this might also affect any jobs using the NIC and Pitch Finding in the NIC though anything is, of course, possible! I was running a OMR workflow on staging and got the following error:
```
RuntimeError: Image view dimensions out of range for data nrows 47 offset_y 511 data nrows 46 data offset_y 512 ncols 56 offset_x 256 data ncols 215 data offset_x 256
```
Which might be from the pitches being off the page, though I'm not sure. I couldn't reproduce on production, as production is failing when the NIC is run ~75% of the time (bug hunt for this in progress, updates to come...).

**Comment by homework36** [comment]
> DDMAL/Neon#1241

Do you know where is that line in Rodan repo to update Neon?

**Comment by yinanazhou** [comment]
> > [DDMAL/Neon#1241](https://github.com/DDMAL/Neon/issues/1241)
> 
> Do you know where is that line in Rodan repo to update Neon?

I just checked the last time we updated Rodan/Neon (#865). Rodan now uses `Neon:develop`. In this case, @JoyfulGen could you confirm that neon staging is in good shape for me to push all the changes to neon production? If so, I would merge the changes.

**Comment by JoyfulGen** [comment]
@yinanazhou, should this issue https://github.com/DDMAL/Neon/issues/1227 be fixed on staging? Or have the changes not been pushed yet? The issue is still happening, but I don't know if that's expected or not...

**Comment by yinanazhou** [comment]
@JoyfulGen This is expected. The changes haven't been merged into verovio. I will have a look at the new one!

**Comment by yinanazhou** [comment]
@JoyfulGen could you please confirm that this issue can be closed?

**Comment by JoyfulGen** [comment]
Confirmed!
